pkg/operator/ceph/object/controller.go

```diff
@@ -456,9 +463,11 @@ func (r *ReconcileCephObjectStore) reconcileCreateObjectStore(cephObjectStore *c
 		if err != nil {
 			return reconcile.Result{}, errors.Wrapf(err, "failed to create object store %q", cephObjectStore.Name)
 		}
+
 	}
 
-	return reconcile.Result{}, nil
+	// Create COSI user and secret
+	return r.reconcileCOSIUser(cephObjectStore)
```

Comment:
The cosi user is being created for both internal and external clusters. @parth-gr @thotz The external cluster would have privileges to create the user, right? I believe so, just wanted to double check.

Reply:
Yes they can also create the user IMO